src/components/replica/index.js

```diff
@@ -0,0 +1,113 @@
+import { Typography } from '@equinor/eds-core-react';
+import * as PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+import AsyncResource from '../async-resource/simple-async-resource';
+import { Code } from '../code';
+import { ReplicaStatus } from '../replica-status';
+import ReplicaSummaryModel from '../../models/replica-summary';
+import { Duration } from '../time/duration';
+import { RelativeToNow } from '../time/relative-to-now';
+import { smallReplicaName } from '../../utils/string';
+import { ReplicaImage } from '../replica-image';
+import { useInterval } from '../../effects/use-interval';
+
+const STATUS_OK = 'Running';
+
+const ReplicaDuration = ({ replica }) => {
+  const [now, setNow] = useState(new Date());
+  useInterval(() => setNow(new Date()), 1000);
+
+  return (
+    <>
+      {replica && (
+        <>
+          <Typography>
+            Created{' '}
+            <strong>
+              <RelativeToNow time={replica.created} />
+            </strong>
+          </Typography>
+          <Typography>
+            Duration{' '}
+            <strong>
+              <Duration start={replica.created} end={now} />
+            </strong>
+          </Typography>
+        </>
+      )}
+    </>
+  );
+};
+
+const ReplicaState = ({ replica }) => (
+  <>
+    {replica && (
+      <>
+        {replica.status !== STATUS_OK && replica.statusMessage && (
+          <>
+            <Typography>Status message is:</Typography>
+            <Code>{replica.statusMessage}</Code>
+          </>
+        )}
+        {!Number.isNaN(replica.restartCount) && replica.restartCount > 0 && (
+          <div>
+            <Typography>Restarted {replica.restartCount} times</Typography>
+          </div>
+        )}
+      </>
+    )}
+  </>
+);
+
+export const Replica = (props) => {
+  const { logState, replica, title, duration, status, state } = props;
```

Comment:
add all expected fields for input props to `Replica.propTypes`, not just `replica`

Reply:
fixed